### PR TITLE
Warning status

### PR DESCRIPTION
**Pull request by liamhuber**
Add a new status 'warning' to jobs, and use it when Vasp faces EDDRMM issues.

### CODE REVIEW

File: pyiron/vasp/base.py [hunk 1]
```diff
@@ -468,24 +468,18 @@ def collect_errors(self):
         """
         num_eddrmm, snap = self._get_eddrmm_info()
 
+        warning_string = "EDDRMM warnings are ignored. EDDRMM occures {} times, first in ionic step {}."
```
Comment (FeLoch) [inline pyiron/vasp/base.py]: shall be "warnings are ignored" be always printed? Even if  `self.get_eddrmm_handling() == "warn"`?

File: pyiron/vasp/base.py [hunk 2]
```diff
@@ -777,14 +771,14 @@ def write_magmoms(self):
...
             s.logger.debug("No magnetic moments")
 
-    def set_eddrmm_handling(self, status="not_converged"):
+    def set_eddrmm_handling(self, status="warn"):
         """
         Sets the way, how EDDRMM warning is handled.
 
         Args:
-            status (str): new status of EDDRMM handling (can be 'not_converged', 'ignore', or 'restart')
+            status (str): new status of EDDRMM handling (can be 'warning', 'ignore', or 'restart')
```
Comment (FeLoch) [inline pyiron/vasp/base.py]: should it be `"warn"` instead of `"warning"`?

File: pyiron/vasp/base.py (hunk 1 above)
Comment (liamhuber) [reply pyiron/vasp/base.py]: Fair. Now it just says "EDDRMM warnings occured {} times, first in ionic step {}"

File: pyiron/vasp/base.py (hunk 2 above)
Comment (liamhuber) [reply pyiron/vasp/base.py]: Yes!

**Comment by liamhuber** [comment]
> Nice work! You may also change the default here:
> 
> https://github.com/pyiron/pyiron/blob/ecf64369dde74305369146dcd2883a346277cf05/pyiron/vasp/base.py#L1706
>
>.

Thanks!

> Also backward compatibility should be conserved in here:
> 
> https://github.com/pyiron/pyiron/blob/ecf64369dde74305369146dcd2883a346277cf05/pyiron/vasp/base.py#L1753
> 
> .

I'm not sure what you mean backwards compatibility in the --- aha, yes, right, got it. Updated.

**Comment by liamhuber** [comment]
@FeLoch No no, you were sufficiently clear, I just had a temporary mental blockage.

**Comment by liamhuber** [comment]
@jan-janssen the Travis error does not seem related to my changes, could you verify please?

**Comment by jan-janssen** [comment]
I restarted the build. You can do the same using the pyiron-runner account and login in on Travis.

**Comment by liamhuber** [comment]
> I restarted the build. You can do the same using the pyiron-runner account and login in on Travis.

Cool, thanks! What exactly is the signal I'm looking for that this the action I ought to take? Is any 'suspicious failure' sufficient? I assume this is not a terribly expensive action.

**Comment by jan-janssen** [comment]
Whenever it says timeout then usually it makes sense to simply restart the tests.

**Pull request merged**